components/form/events.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Events
+page_title: Form Events
+description: Form for Blazor - Events.
+slug: form-events
+tags: telerik,blazor,form,edit,events
+published: True
+position: 25
+---
+
+# Form Events
+
+The Form component for Blazor exposes events that allow you to respond to user actions and provide custom logic.
+
+* [OnSubmit](#onsubmit)
+* [OnValidSubmit](#onvalidsubmit)
+* [OnInvalidSubmit](#oninvalidsubmit)
+
+## OnSubmit
+
+The `OnSubmit` event fires when the user clicks on the Submit button in the Form. It takes as a parameter the `EditContext` object and is used to trigger some custom logic based on the validity of the form. When this event is setup the `OnValidSubmit` and `OnInvalidSubmit` events will not be fired.
+  
+The `OnSubmit` event is mapped to the `OnSubmit` event of the <a target="_blank" href="https://docs.microsoft.com/en-us/aspnet/core/blazor/forms-validation?view=aspnetcore-5.0#built-in-forms-components">Microsoft EditForm</a>
+
+
+>caption Handle the OnSubmit event
+
+````CSHTML
+@* Use the OnSubmit event to trigger some custom logic depending on the validity of the form *@
+
+@using System.ComponentModel.DataAnnotations 
+
+<TelerikForm EditContext="@myEditContext"
```

Comment:
Check if you need to provide an edit context here. I think it should not be necessary, the form would create one based on the model. If it is, mention this explicitly. If it isn't - tweak this example or make a new one where the first one uses the model only.

Reply:
we do not, as we discussed yesterday Its just used as an example